From: David Faure <faure@kde.org>
Date: Sun, 1 Jan 2017 13:16:10 +0100
Subject: ForwardingSlaveBase: fix passing of Overwrite flag to kio_desktop

Unittest will be committed to plasma-workspace/kioslave/desktop later.

BUG: 360487
FIXED-IN: 5.30
---
 src/core/forwardingslavebase.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/core/forwardingslavebase.cpp b/src/core/forwardingslavebase.cpp
index 5e724764..0a16c2ef 100644
--- a/src/core/forwardingslavebase.cpp
+++ b/src/core/forwardingslavebase.cpp
@@ -273,7 +273,7 @@ void ForwardingSlaveBase::symlink(const QString &target, const QUrl &dest,
 
     QUrl new_dest;
     if (d->internalRewriteUrl(dest, new_dest)) {
-        KIO::SimpleJob *job = KIO::symlink(target, new_dest, flags & HideProgressInfo);
+        KIO::SimpleJob *job = KIO::symlink(target, new_dest, flags | HideProgressInfo);
         d->connectSimpleJob(job);
 
         d->eventLoop.exec();
